### DISCUSSION TITLE
Possible annotation interfaces and devices for MONAI Label

### DISCUSSION THREAD

**Discussion by Zrrr1997** [Project-MONAI/MONAILabel#1411] [General]
Dear MONAI Label,

thank you for this wonderful project. I was wondering what devices are usually used when annotating data with MONAI Label. The way I understand it, MONAI Label support Windows and Ubuntu, so the backend server you connect to, e.g., with 3D Slicer must be with on of these operating systems.

However, I was wondering whether it is possible to integrate some other tactile interface, e.g., a touchscreen on a tablet, when annotating volumes with 3D Slicer. There are some demos on how Slicer runs on Windows-enabled tablets:
https://www.youtube.com/watch?v=0Nsya9vBswE 
https://www.youtube.com/watch?v=QQ9kFT8XhtA  

I was wondering whether this would also be possible with the MONAI Label plugin and whether someone has experience with annotating volumes with, e.g., a stylus. 

3D Slicer supports macOS, Linux, and Windows so I assume the tablet should run those. I would be happy to hear what tablet-based setups are possible with MONAI Label :) 

Best,
Zdravko

**Comment by lassoan** [comment]
You can run MONAILabel in Slicer on a touchscreen/pen tablet, running on Windows, Linux, macOS. Windows, and in particular the Microsoft Surface product line is the farthest ahead in pen and touch technology for personal computers, so if you want to get the most robust pen and touch experience I would recommend a Surface tablet/laptop/desktop computer.

Some people prefer to use a drawing tablet (stylus is off the screen, on a digitizer board), which generally works well, too, but it is a bit harder to guarantee optimal performance since there is a greater variety of tablets and their operating system integration is not always seamless. For some devices you may need to tune tablet settings and some advanced options in Slicer.

**Comment by diazandr3s** [comment]
Hi @Zrrr1997,

For a good user experience, we recommend using MONAI Label on a GPU-enabled computer. MONAI Label is being tested on Linux and Windows.
You could connect the 3D Slicer running on a tablet to the MONAI Label server running on a computer with GPU(s).

I hope this helps,
